CONTRIBUTING.md

```diff
@@ -0,0 +1,50 @@
+# Contributing Guide
+
+## Initial Setup
+
+To begin contributing to mCodingBot, you must first fork the repository. After forking, clone:
+
+```
+git clone <url to your fork>
+cd mCodingBot
+```
+
+Next, you need to set up poetry.
+
+```
+curl -sSL https://install.python-poetry.org | python3 -
+poetry install
+```
+
+See the [installation guide](https://python-poetry.org/docs/#installation) for alternative methods.
+
+## Running the Bot
+
+First, if you're making changes that require the database, you will need to set up Postgres. Once installed, create a user named `mcodingbot`, and then a database called `mcodingbot` that is owned by that user.
+
+To run the bot, you first need to set up your `config.json` file. Look at [config.json.example](https://github.com/mcb-dev/mCodingBot/blob/main/config.json.example) for an example configuration. Make sure to set `no_db_mode: true` if you chose not to set up Postgres. If you did set up Postgres, you only need to put your password (if any) in the config.json.
+
+Once the config is setup, you can run the bot with `poetry run python3 -m mcodingbot`.
+
+## Contributing
+
+To create a pull request to mCodingBall, first create a new branch for your changes.
```

Comment:
```suggestion
To create a pull request to mCodingBall, first, create a new branch for your changes.
```

Reply:
This feels wrong to me because it makes "first" look like a parenthetical. Imo the best thing to do here is reword the entire sentence if you aren't happy with it.